documentation/contributing/styleguide.adoc

```diff
@@ -127,15 +127,15 @@ Learn more about reusing content in the link:https://redhat-documentation.github
 [[replaceables]]
 == Styling replaceables
 
-Replaceable values in the text and code must be styled as monospace, italics, lowercase, arrow brackets, and hyphens.
+Replaceable values in the text and code should be styled in `monospace`, with italics, uppercase, and hyphens.
```

Comment:
Is this intentionaly in this PR? It seems to be unrelated so I'm not sure.

Reply:
It's half-related. We're starting to apply a new _replaceables_ format to the doc, which I used on the new oauth config. So I thought I would update the directions too.